.github/actions/build-push-multi-arch/action.yml

```diff
@@ -0,0 +1,78 @@
+name: 'Build and Push (Multi-Arch)'
+description: 'Build a Docker image for amd64 and/or arm64 and push it to GitHub Container Registry'
+author: aica-technology
+branding:
+  icon: file-plus
+  color: gray-dark
+inputs:
+  architecture:
+    description: 'The desired architecture [`amd64` or `amd64 & arm64`]'
+    required: true
+    default: 'amd64 & arm64'
+  path:
+    description: 'Path to the directory that specifies the Docker build context'
+    required: true
+    default: '.'
+  filename:
+    description: 'The full name of the Dockerfile with respect to the build context'
+    required: true
+    default: 'Dockerfile'
+  image_name:
+    description: 'The name of the image (for example "aica-technology/image-name")'
+    required: true
+  image_tag:
+    description: 'The tag of the image (for example "latest")'
+    required: true
```

Comment:
nitpick: I would prefer to see these required inputs with no default value at the top, then required inputs with default values, then non-required inputs. Just seems clearer to me. (the secret is an exception of course because it is more of a technicality)

Reply:
Done :+1:

Reply:
In retrospect, it's a bit strange to me that there are inputs with default values that are still required. Seems like the default value has no effect in that case.

Reply:
I guess that's true yeah. I will make a backlog issue for when we come back to this at some point.